### ISSUE TITLE
AttributeError: 'datetime.date' object has no attribute 'minute'

### ISSUE THREAD

**Issue by yianjiajia** [sphinx-doc/sphinx#2365]
```
  4 # Jinja2 version: 2.8
  5 # Last messages:
  6 #   ../TESTING.rst:279: SEVERE: Title level inconsistent:
  7 
  8 Adding more tests
  9 ~~~~~~~~~~~~~~~~~
 10 #   ../TESTING.rst:67: ERROR: Unknown target name: "test".
 11 #   ../TESTING.rst:73: ERROR: Unknown target name: "test".
 12 #   looking for now-outdated files...
 13 #   none found
 14 #   pickling environment...
 15 #   done
 16 #   checking consistency...
 17 #   done
 18 #   preparing documents...
 19 # Loaded extensions:
 20 #   sphinx.ext.graphviz (1.4a1) from /usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/ext/graphviz.pyc
 21 #   sphinx.ext.autodoc (1.4a1) from /usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/ext/autodoc.pyc
 22 #   oslosphinx (unknown version) from /usr/lib/python2.7/site-packages/oslosphinx-4.3.0-py2.7.egg/oslosphinx/__init__.pyc
 23 #   alabaster (0.7.7) from /usr/lib/python2.7/site-packages/alabaster-0.7.7-py2.7.egg/alabaster/__init__.pyc
 24 #   sphinx.ext.coverage (1.4a1) from /usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/ext/coverage.pyc
 25 #   sphinx.ext.pngmath (1.4a1) from /usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/ext/pngmath.pyc
 26 #   sphinx.ext.todo (1.4a1) from /usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/ext/todo.pyc
 27 #   sphinx.ext.ifconfig (1.4a1) from /usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/ext/ifconfig.pyc
 28 Traceback (most recent call last):
 29   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/cmdline.py", line 244, in main
 30     app.build(opts.force_all, filenames)
 31   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/application.py", line 286, in build
 32     self.builder.build_update()
 33   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/builders/__init__.py", line 251, in build_update
 34     'out of date' % len(to_build))
 35   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/builders/__init__.py", line 322, in build
 36     self.write(docnames, list(updated_docnames), method)
 37   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/builders/__init__.py", line 349, in write
 38     self.prepare_writing(docnames)
 39   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/builders/html.py", line 296, in prepare_writing
 40     language=self.config.language)
 41   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/util/i18n.py", line 177, in format_date
 42     return babel_format_date(date, format, locale=language)
 43   File "/usr/lib/python2.7/site-packages/Sphinx-1.4a1-py2.7.egg/sphinx/util/i18n.py", line 156, in babel_format_date
 44     return babel.dates.format_date(date, format, locale=locale)
 45   File "/usr/lib/python2.7/site-packages/babel/dates.py", line 568, in format_date
 46     return pattern.apply(date, locale)
 47   File "/usr/lib/python2.7/site-packages/babel/dates.py", line 891, in apply
 48     return self % DateTimeFormat(datetime, locale)
 49   File "/usr/lib/python2.7/site-packages/babel/dates.py", line 888, in __mod__
 50     return self.format % other
 51   File "/usr/lib/python2.7/site-packages/babel/dates.py", line 941, in __getitem__
 52     return self.format(self.value.minute, num)
 53 AttributeError: 'datetime.date' object has no attribute 'minute'
```

**Comment by tk0miya** [comment]
Thank you for reporting.
Please let me know about your configurations:
- language
- today_fmt
- html_last_updated_fmt

**Comment by tk0miya** [comment]
Maybe this fixed at d35ccb9.
If you have a trouble again please reopen.

Thanks,
